dist/dist_test.go

```diff
@@ -105,3 +105,46 @@ func TestPushManifest(t *testing.T) {
 		t.Fatalf("got an unexpected reply: %v", err)
 	}
 }
+
+func TestListRepos(t *testing.T) {
+	reqPath := "catalog"
+
+	regAuthCtx.Scope.RemoteName = reqPath
+	regAuthCtx.Scope.Actions = "pull"
+
+	indexServer := auth.GetIndexServer(regURL)
+
+	// NOTE: it will fail when testing against docker.io, as '/v2/_catalog' endpoint
```

Comment:
This doesn't make sense to test in the context of OCI. `_context` isn't a standard path.

Reply:
As far as I understand from the [spec](https://github.com/opencontainers/distribution-spec/blob/0a02fd56fccc22f8033d766e91ccf7b452a1caa6/spec.md#listing-repositories), `GET /v2/_catalog` indeed seems to be in the spec.

Do you mean that the listing repo part of the spec needs to be changed?
Or do you mean that the spec is correct, but tests should not take into account that part of the spec, because it's implementation specific?

BTW I see this code has a bug. I'll fix it soon.